src/telemetry/metrics.yaml

```diff
@@ -168,6 +168,39 @@ popup:
         description: Which tier of plan the user is on [Free, Plus]
         type: string
 
+banner:
```

Comment:
I'm assuming this doesn't need another data review before merging?

Reply:
Since this event module is recording the same telemetry like for example `page` I think we were not sure about that. I’ll ask the Glean team so we have clarity on that before merging.

cc: @rhelmer

Reply:
We are good to go with this event as the intent and use of `banner.view` is already covered by the original data review. Upon [feedback](https://matrix.to/#/!XDoIOuMgPLgJFSLqRP:mozilla.org/$akGWGnvckAaxwiYd9-7Te9wpIr1upGnmeM6nCxoJXV4?via=mozilla.org&via=matrix.org&via=t2bot.io) from a data steward on Matrix, I’ve added a [comment to the original data review](https://bugzilla.mozilla.org/show_bug.cgi?id=1823766#c15) as a future reference.

cc: @rhelmer